### ISSUE TITLE
Publish the library to a public maven repository

### ISSUE THREAD

**Issue by tkirshboim** [libpd/pd-for-android#20]
That repository will probably be [jcenter](https://bintray.com/bintray/jcenter)

**Comment by tkirshboim** [comment]
As an maven artifact id me and @joebowbeer thought of using:
```
org.puredata.android:pd-core:1.0.0
```

Pinging @nettoyeurny @chr15m @danomatika - any thought on this?

**Comment by nettoyeurny** [comment]
I think libpd is potentially ambiguous because it sometimes refers to the
underlying C library and sometimes to the platform-specific bits as well.
Since the platform-specific project for Android is called PdCore, it makes
a lot of sense to call the maven artifact pd-core.

**Comment by tkirshboim** [comment]
Status: published an RC1 version on Bintray - waiting for approval to be added to jcenter:
https://bintray.com/pd-for-android/maven/pd-for-android/1.0.0-rc1/view

**Comment by tkirshboim** [comment]
JCenter approved the request. pd-for-android is now publicly available as an Android library. :tada: 
I'm leaving this ticket open until we've updated the documentation with explanation how to use the library this way. But briefly- what was until now:

```
dependencies {
    compile project(':PdCore')
}
```
can be replaced with :

```
dependencies {
    compile 'org.puredata.android:pd-core:1.0.0-rc1'
}
```
(assuming that the project uses jcenter to resolve maven dependencies)

**Comment by deepkanwal** [comment]
I'm getting the following error when attempting a Gradle sync using `compile 'org.puredata.android:pd-core:1.0.0-rc1'` with a new empty project.

`Error:Failed to resolve: com.github.nettoyeurny:btmidi:9bc54ccbe1`

Any ideas?

**Comment by tkirshboim** [comment]
@deepkanwal looks like I was too eager to celebrate.. The RC1 version requires the jitpack repository, which is not what we want. I didn't notice that I had it included in my test setup.
I'll fix this and update the issue here.

**Comment by tkirshboim** [comment]
@deepkanwal I've updated the version and tested it locally and it seems to work. I'm going to test it more in the next days, but if you can, please try it again with the gradle task ```assembleRelease --refresh-dependencies``` to make sure that the pd-for-android dependency is not taken from the gradle cache. Thanks!

**Comment by tkirshboim** [comment]
@deepkanwal I have fixed this and released a new version that you can use with:
```
dependencies {
    compile 'org.puredata.android:pd-core:1.0.0-rc2'
}
```
Let me know if this for some reason still doesn't work for you. Thanks!

**Comment by tkirshboim** [comment]
Update: [We still have some issues with publishing the library on JCenter](https://github.com/libpd/pd-for-android/pull/25#issuecomment-160414876). The 'rc2' release mentioned above might work for you, but until we have AndroidMidi published on Jcenter as well it's probably better to use Jitpack as described here:
https://jitpack.io/#libpd/pd-for-android/v1.0.0-rc3

**Comment by luizmello** [comment]
Hi, I just test the Jitpack (rc3) repository and it works with no errors.
You are doing a great job here!
thanks a lot for creating this method with repositories, this is much faster and simple.

**Comment by tkirshboim** [comment]
@luizmello thanks! Glad to hear it works :)

**Comment by tkirshboim** [comment]
I just released version '1.0.0-rc3' on Jcenter which uses AndroidMidi from JCenter as well:
https://bintray.com/pd-for-android/maven/pd-for-android/1.0.0-rc3/view

Please try it out if you have a moment and see #27 for more details. Thanks!
